server/.env.development

```diff
@@ -0,0 +1,13 @@
+# DO NOT ADD SECRETS TO THIS FILE. This is a good place for defaults.
+# If you want to add secrets use `.env.development.local` instead.
+
+# for local dev, we turn off auth, unless specifically working on auth
+BYPASS_AUTH=true
+DUMMY_EMAIL="demo@draft.test"
+DUMMY_PASSWORD="password"
```

Comment:
@akoshelev while this is still checked in, it's now in the `.env.development`, which will never be used in production. it's also no longer in `seed.sql`.